Gruntfile.js

```diff
@@ -5,9 +5,10 @@ module.exports = function(grunt) {
       options: {
         includePaths: [
           "node_modules/bourbon/core",
-          "node_modules/bourbon-neat/app/assets/stylesheets",
-          "node_modules/normalize.css",
+          "node_modules/bourbon-neat/core",
+          "node_modules/breakpoint-sass/stylesheets",
```

Comment:
I was hoping that we could avoid the addition of this library. Is this a stop gap with another round of refactoring coming or something that is absolutely needed?